src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+                

```

Comment:
Trailing whitespaces...

Reply:
Yep. Turns out that Visual Studio will automatically add those trailing spaces to a .resw file pretty much always. That's why every other commit in this repo removes them, then adds them back, depending on what editor the author is using. It really doesn't matter in the end though.

Reply:
(thanks for the review though? :P)